themes/Arch-Colors/lxqt-panel.qss

```diff
@@ -179,7 +179,128 @@ QMenu::indicator:non-exclusive:checked {
 
 #MainMenu ActionView::item:selected {
     background: #1899da;
-    color: white
+    color: white;
+}
+
+/*
+ * Fancymenu
+ */
+#FancyMenu {
+    qproperty-icon: url(mainmenu.svg);
+    background: #162635;
+    margin: 0px;
+    padding: 3px;
+    color: #e7eff1;
+}
+
+LXQtFancyMenuWindow{
+    background: #041b27;
+}
+
+#FancyMenu:hover {
+    color: #d7e3e6;
+}
+
+#FancyMenu  #AppView {
+    background: #06293a;
+    border: 1px solid grey;
+}
+
+#FancyMenu QListView::item {
+    color: #d7e3e6;
+    padding: 5px 30px 5px 3px;
+}
+
+#FancyMenu QListView::item:hover {
+    background: #003d5b;
+    color: #d7e3e6;
+    padding: 0px;
+}
```

Comment:
IMO, shifting to right on hovering is a little disturbing. Is it intentional? If not, please change the above two blocks to these (also a 30-px right padding seems unnecessary, but you could keep it if you like it):

```xml
#FancyMenu QListView::item {
    color: #d7e3e6;
    padding: 5px 3px 5px 3px;
    border: 0px;
}

#FancyMenu QListView::item:hover {
    background: #003d5b;
    color: #d7e3e6;
}
```

Reply:
> This padding: 0px was an illogical hack

Haha... CSS sometimes seems illogical, yes. However, it's a matter of inheritance: when something isn't specified, it might be inherited from the Qt style and cause confusion.

Reply:
btw I just  copied the themes from mainmenu and replaced `Main → Fancy` and ` Action → QList` and removed usused stuff.

Reply:
Not all widgets get similar details from the Qt style, although their names may be the same in CSS — Qt's code is quite complicated. However, please check if the main menu has a similar problem.

In short, if you see an "unexplainable" effect with CSS, you could first search for another part of CSS that sets a value and then, if there isn't any, search for totally missing values.

Reply:
> if there isn't any, search for totally missing values

that was the point, but it's still illogical ;)